packages/base/links-to-many-component.gts

```diff
@@ -195,7 +200,13 @@ class LinksToManyStandardEditor extends GlimmerComponent<LinksToManyStandardEdit
                 )
                 as |Item|
               }}
-                <Item @format='fitted' />
+                <button
+                  type='button'
+                  class='card-button'
+                  {{on 'click' (fn this.viewCard boxedElement.value)}}
+                >
+                  <Item @format='fitted' />
+                </button>
```

Comment:
It was not this way in the past because we conceived of this behavior as something that interact mode added to card rendering. Are we clear that we want to change that? This may have far-reaching impact.